### DISCUSSION TITLE
How to use templates?

### DISCUSSION THREAD

**Discussion by mattansb** [quarto-dev/quarto-cli#5367] [Q&A] [answered]
***Assuming in am averse to using the CLI to quarto***, how do I actually install and use templates?

It seems like they can only be installed using the CLI. So far, I've run:

```
quarto install extension schochastics/quarto-sketchy-html
```

Which executed successfully.

I've then created an new `.qmd` file with the following head:

```
---
format: sketchy-html
---
```

But am getting:
```
ERROR: Unable to read the extension 'quarto-sketchy'.
Please ensure that you provided the correct id and that the extension is installed.
```

What am I missing?

I'm working in RStudio 2023.03.0, R 4.2.2, Quarto 1.2.335

**Comment by mcanouil** [comment]
Did you run the command inside the directory where your document lives?
Extensions are installed locally, thus you'll need to install them in each of your Quarto projects.

**Reply by mattansb** [reply]
I did not :/

Is there a way to do this without the CLI? Preferably in R/RStudio?

**Reply by mcanouil** [reply, marked as answer]
> I did not :/

Thus, the result was expected.

> Is there a way to do this without the CLI? Preferably in R/RStudio?

Not currently, but maybe RStudio will take care of that in the future.
Note that in RStudio when inside a project, you can open the integrated terminal and simply type `quarto add <owner>/<repository>` which will add the extension in the root of your RStudio project (`quarto install extension` is kind of the old syntax).

**Reply by mattansb** [reply]
Thank you very much!
